app/src/main/java/com/keylesspalace/tusky/viewdata/StatusViewData.java

```diff
@@ -58,8 +58,8 @@ private StatusViewData() {
         private final String nickname;
         private final String avatar;
         private final Date createdAt;
-        private final String reblogsCount;
```

Comment:
I am not sure about this one. For me ViewData is about minimum representation required to render a thing. I think it's views responsibility to decide how it wants to represent it. Is there a reason for this change?

Reply:
I don't understand why numbers where represented as String, does not make sense to me. 
Yes, I added number formatting in the views, looks much nicer now.

Reply:
Oh, crap, sorry, it got reversed in my head, you've done it right.